### PR TITLE
Post release fixes

### PR DESCRIPTION
**Pull request by jameschristopher**

### CODE REVIEW

File: alphalens/performance.py [hunk 1]
```diff
@@ -39,10 +39,10 @@ def factor_information_coefficient(factor,
     forward_returns : pd.DataFrame - MultiIndex
         Forward returns in indexed by date and asset.
         Separate column for each forward return window.
-    sector_adjust : bool
-        Demean forward returns by sector before computing IC.
-    by_sector : bool
-        If True, compute daily IC separately for each sector.
+    group_adjust : bool
```
Comment (twiecki) [inline alphalens/performance.py]: How do I pass in a custom group?

Comment (jameschristopher) [inline alphalens/performance.py]: At this point we assume you've already added the group to the MultiIndex. This isn't mentioned in the docstring and should be.

**Comment by twiecki** [comment]
image

Seems like there's a problem with the sector codes.

**Comment by jameschristopher** [comment]
Hmmm. Interesting. It's like that on master too. It doesn't seem to be a problem when running the same code on Quantopain's iPython environment. https://www.quantopian.com/posts/alphalens-a-new-tool-for-analyzing-alpha-factors

Perhaps this is environment specific.

**Comment by twiecki** [comment]
:+1:

**Pull request merged**